hw2/binding.gyp

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
If you build and install the shared library together with its public header at the standard paths of the operating system, then you can link it to the addon easily e.g.
```javascript
{
    'targets': [{
        'target_name': 'node-mymath',
        'sources': ['src/binding.cxx'],
        'include_dirs': [
            '<!@(node -p "require(\'node-addon-api\').include")'
        ],
        'dependencies': [
            '<!(node -p "require(\'node-addon-api\').gyp")'
        ],
        'cflags_cc!': ['-fno-exceptions'],
        'link_settings': {
            'libraries': ['-lmymath']
        }
    }]
}
```